src/lib/Libwin/accesinfo.c

```diff
@@ -181,25 +185,39 @@ secure_file(char *path, char *user, ACCESS_MASK mask)
 	ACL	*pdacl;
 	struct  stat sbuf;
 	SECURITY_INFORMATION	si = 0;
-	char	logb[LOG_BUF_SIZE] = {'\0' } ;
 	char	*gname = NULL;
 
-	if (path == NULL || *path == '\0')
+	if (path == NULL || *path == '\0') {
+		log_eventf(PBSEVENT_ERROR, 0, LOG_DEBUG, __func__, "invalid path");
 		return (0);
+	}
 
-	if (lstat(path, &sbuf) == -1)
+	if (lstat(path, &sbuf) == -1) {
+		log_eventf(PBSEVENT_ERROR, 0, LOG_DEBUG, __func__, "Path don't exists");
```

Comment:
We should pass on the errno here

Reply:
If lstat() fails then the corresponding implementation in util.c logs the error code and message.
Do you recommend to remove this log message from here?